### ISSUE TITLE
Multiple objects detected with real-time bounding-box

### ISSUE THREAD

**Issue by JakeMalis** [giacomobartoli/vision-kit#2]
Hello!

How could I make multiple objects (i.e. a Pikachu and a Mewtwo) have bounding boxes around them in real-time, rather than have it take a picture when an object is detected and then apply the bounding box?

Thanks,
Jake

**Comment by giacomobartoli** [comment]
Clone this repo and place the files into /home/pi/AIY-projects-python/src/examples/vision .
After that, type ./custom_pikachu_detector from a shell.
Make sure your Vision Kit is updated to the last version.

Concerning the detection, this part depends on the frozen graph. The frozen graph included into this repo (.pb file) is made for detecting only Pikachu. If you want to detect other objects you must train another neural network from scratch.

**Comment by JakeMalis** [comment]
I haven't had a chance to test this yet but will it do real-time (I understand there will be a slight delay) bounding boxes if there are multiple pikachus? Or will it take a photo and then put a bounding box?

**Comment by giacomobartoli** [comment]
Yes, this code works in real time. Even detecting multiple objects.

**Comment by giacomobartoli** [comment]
It means that when Pikachu is detected, the Vision Kit take a photo automatically.
You don't have to change it.

**Comment by JakeMalis** [comment]
I thought that was causing this error

> Traceback (most recent call last):
  File "./custom_object_detection.py", line 127, in <module>
    main()
  File "./custom_object_detection.py", line 85, in main
    detections = enumerate(object_detection_helper.get_objects(result, 0.3))
  File "/opt/aiy/projects-python/src/examples/vision/custom_object_detection/object_detection_helper.py", line 219, in get_objects
    score_threshold, size, offset)
  File "/opt/aiy/projects-python/src/examples/vision/custom_object_detection/object_detection_helper.py", line 71, in _decode_detection_result
    assert len(logit_scores) == _NUM_LABELS * _NUM_ANCHORS
AssertionError

**Comment by giacomobartoli** [comment]
The error is generated at line 71, which is an AssertionError.
Try to delete or comment out `assert len(logit_scores) == _NUM_LABELS * _NUM_ANCHORS`

**Comment by JakeMalis** [comment]
I just realized that for the _NUM_LABELS variable it is the amount of objects plus the background.

Let me try again.

**Comment by giacomobartoli** [comment]
@JakeMalis if you're able to run this project correctly I would close this issue.
